test/Common/DurableClientBaseTests.cs

```diff
@@ -142,6 +144,91 @@ public async Task TerminateAsync_NonRunningOrchestrator_ThrowsException()
         }
 
 #if !FUNCTIONS_V1
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task DurableClient_DI_StartNewAsync_ReturnsInstanceId()
```

Comment:
nit: I don't think the names of these tests are quite accurate, as it implies we are testing the full `IDurableClientFactory` code path, which we aren't quite doing. I would maybe mark them as "DurableClient_ExternalApp_*` instead. It's not perfect, but a bit more accurate.